spinalcordtoolbox/utils.py

```diff
@@ -430,6 +453,46 @@ def sct_progress_bar(*args, **kwargs):
     return tqdm.tqdm(*args, **kwargs)
 
 
+def zipdir(path_to_zip, fname_zip='archive.zip'):
+    """
+    Create zip archive.
+    :param path_to_zip: file or directory
```

Comment:
The interface can be improved to manage on-the-fly archiving of several directories.

Reply:
yup, i'll work on it

Reply:
fixed in 0407fc2fc5b2782080d62e801e42d94c76996279

Reply:
I think this duplicates the functionality of `shutil.make_archive` https://stackoverflow.com/a/25650295/6227376